addon/-private/properties/blurrable.js

```diff
@@ -0,0 +1,75 @@
+import { assign } from '../helpers';
+import { getExecutionContext } from '../execution_context';
+
+/**
+ *
+ * Blurs element matched by selector.
+ *
+ * @example
+ *
+ * // <input class="name">
+ * // <input class="email">
+ *
+ * const page = PageObject.create({
+ *   blur: blurrable('.name')
+ * });
+ *
+ * // blures on element with selector '.name'
+ * page.blur();
+ *
+ * @example
+ *
+ * // <div class="scope">
+ * //   <input class="name">
+ * // </div>
+ * // <input class="email">
+ *
+ * const page = PageObject.create({
+ *   blur: blurrable('.name', { scope: '.scope' })
+ * });
+ *
+ * // blures on element with selector '.scope .name'
+ * page.blur();
+ *
+ * @example
+ *
+ * // <div class="scope">
+ * //   <input class="name">
+ * // </div>
+ * // <input class="email">
+ *
+ * const page = PageObject.create({
+ *   scope: '.scope',
+ *   blur: blurrable('.name')
+ * });
+ *
+ * // blures on element with selector '.scope .name'
+ * page.blur();
+ *
+ * @public
+ *
+ * @param {string} selector - CSS selector of the element which will be blurred
+ * @param {Object} options - Additional options
+ * @param {string} options.scope - Nests provided scope within parent's scope
+ * @param {number} options.at - Reduce the set of matched elements to the one at the specified index
+ * @param {boolean} options.resetScope - Ignore parent scope
+ * @param {string} options.testContainer - Context where to search elements in the DOM
+ * @param {string} options.eventProperties - Event properties that will be passed to trigger function
```

Comment:
Looks like `eventProperties` are not used in this helper

Reply:
ah good point, yeah it's not really something you can/would want to pass along. I'll remove it from the docs